src/controllers/movies/moviesrecommended.js

```diff
@@ -114,19 +114,26 @@ define(["events", "layoutManager", "inputManager", "userSettings", "libraryMenu"
         var allowBottomPadding = true;
 
         if (enableScrollX()) {
-            allowBottomPadding = false;
-            html += '<div is="emby-itemscontainer" class="itemsContainer hiddenScrollX padded-left padded-right">';
+            html += '<div is="emby-scroller" class="padded-top-focusscale padded-bottom-focusscale" data-mousewheel="false" data-centerfocus="true">';
+            html += '<div is="emby-itemscontainer" class="itemsContainer scrollSlider focuscontainer-x">';
         } else {
-            html += '<div is="emby-itemscontainer" class="itemsContainer vertical-wrap padded-left padded-right">';
+            html += '<div is="emby-itemscontainer" class="itemsContainer focuscontainer-x padded-left padded-right vertical-wrap">';
         }
 
         html += cardBuilder.getCardsHtml(recommendation.Items, {
             shape: getPortraitShape(),
             scalable: true,
             overlayPlayButton: true,
-            allowBottomPadding: allowBottomPadding
+            allowBottomPadding: allowBottomPadding,
+            showTitle: true,
+            showYear: true,
+            centerText: true
         });
-        html += "</div>";
```

Comment:
I think this was used to close the `verticalSection` div element on line 112, so we need to keep it.

Reply:
> 
> 
> I think this was used to close the `verticalSection` div element on line 112, so we need to keep it.

the verticalSection was closed at line 137

Reply:
Doesn't 134 close the `itemsContainer` and 137 close the scroller?